concerned-coyotes/earlyinternet/weather/utils.py

```diff
@@ -1,21 +1,29 @@
-import json
-import urllib.request
+import requests
+from datetime import datetime
 
 
-
-def get_weather(lat,lon):
-    key='0a2c03e9dee2533389d7b459d0f8595e'
-    # source contain JSON data from API 
-    source=urllib.request.urlopen(f'http://api.openweathermap.org/data/2.5/weather?lat={lat}&lon={lon}&appid={key}').read()
-
-    # converting JSON data to a dictionary
-    list_of_data=json.loads(source)
-    data = {  
-            "Temp": str(float(list_of_data['main']['temp'])-272.15)+ '°C',
-            "City":str(list_of_data['name']), 
-        } 
+def get_weather(lat: float, lon: float) -> dict:
+    """
+    Take latitude and longitude to get weather data
+    using openweather api and return data in dict format
+    """
+    weather_key = '0a2c03e9dee2533389d7b459d0f8595e'
+    # source contain JSON data from API
+    source = requests.get(
+        f'http://api.openweathermap.org/data/2.5/weather?lat={lat}&lon={lon}&appid={weather_key}')
+    # list_of_data contain json encoded content
+    list_of_data = source.json()
+    data = {"weather": {
+        "celsius": int(list_of_data['main']['temp'])-272,
+        "fahrenheit": int(((list_of_data['main']['temp'])-272)*1.8+32)},
```

Comment:
The temp value is Fahrenheit by default no need to convert to celsius and then back to fahrenheit

Reply:
I think the default is kelvin that's why I convert it

Reply:
You are right my bad